packages/signer-cli/src/signer.ts

```diff
@@ -1,8 +1,11 @@
 // Copyright 2018-2020 @polkadot/signer-cli authors & contributors
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Would really prefer not doing this here or in the other file, trying to rid the code of any `any` not add more  - rather assuming parsed data as string. (Or if you want to be extensive, add the types that are allowed)